sphinxcontrib/drawio/__init__.py

```diff
@@ -250,6 +250,10 @@ def _drawio_export(self, input_abspath, options, out_filename):
         if builder.config._display:
             new_env["DISPLAY"] = ":{}".format(builder.config._display)
 
+        if ('ELECTRON_RUN_AS_NODE' in new_env):
+            # This env var will cause the drawio desktop app to not run properly
+            new_env.pop("ELECTRON_RUN_AS_NODE")
```

Comment:
Perhaps consider not using an `if` and instead using `new_env.pop("ELECTRON_RUN_AS_NODE", None)`.